content/influxdb/v2.0/reference/config-options.md

```diff
@@ -1469,13 +1470,63 @@ session-renew-disabled = true
 
 ---
 
+
+### sqlite-path
+
+Specify a user-defined path to the SQLite database file.
+
+| influxd flag    | Environment variable  | Configuration key |
+|:----------------|:----------------------|:------------------|
+| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
+
+**Default:** If not set explicitly, the default path for the SQLite database file
+is in the [`INFLUXD_BOLT-PATH`](#bolt-path) directory as `influxd.sqlite`.
```

Comment:
```suggestion
Path to the SQLite database file.

**Default:** `~/.influxdbv2/influxd.sqlite`

| influxd flag    | Environment variable  | Configuration key |
|:----------------|:----------------------|:------------------|
| `--sqlite-path` | `INFLUXD_SQLITE_PATH` | `sqlite-path`     |
```

Reply:
See https://github.com/influxdata/influxdb/pull/21543/files#diff-51b7cb5da4f014baef4fe7492c9242fe230f76a48e3b798c28ae92ab4f21db61R292-R293 and resolved conversation there. It's strictly correct according to the code that the default for `sqlite-path` uses the same directory as `bolt-path`, which could be somewhere other than `~/.influxdbv2/influxd.sqlite`. I think it's better to be strict about this behavior and risk some confusion unless/until this code changes.